src/DeviceInterface.js

```diff
@@ -349,10 +394,12 @@ class AppleDeviceInterface extends InterfacePrototype {
         /**
          * Gets a single identity obj once the user requests it
          * @param {Number} id
-         * @returns {APIResponse<IdentityObject>}
+         * @returns {Promise<{success: IdentityObject}>}
          */
-        this.getAutofillIdentity = (id) =>
-            wkSendAndWait('pmHandlerGetIdentity', { id })
+        this.getAutofillIdentity = (id) => {
```

Comment:
I've changed this to use the locally stored identities since we format them upon receipt. Note that with Jonathan's work this will probably become the standard for all the data.